client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -118,8 +129,8 @@ export function SitesDashboard( { queryParams: { search, status } }: SitesDashbo
 										defaultValue={ search }
 									/>
 								</SearchWrapper>
-								{ filteredSites.length > 0 ? (
-									<SitesTable sites={ filteredSites } />
+								{ sortedSites.length > 0 ? (
```

Comment:
From a product perspective, sorting by `updatedAt` is much less useful than having my primary and most recently visited sites at the top of the list.

Maybe we should make sorting by `updatedAt` opt-in behavior?

Reply:
We wanted to sort by Last Publish in the initial iteration (https://github.com/Automattic/wp-calypso/issues/65168).

Letting a user decide which field we should use to sort (e.g., by clicking on column names), what should be the sort order (ascending or descending), and what should be the default sort order makes sense, but I think it deserves the separate research and discussion. What do you think?

Reply:
> What do you think?

I still think: _From a product perspective, sorting by `updatedAt` is much less useful than having my primary and most recently visited sites at the top of the list._

I'd be fine to land this PR as-is, though.

Reply:
[I feel the same](https://github.com/Automattic/wp-calypso/issues/65168#issuecomment-1196518433), but agree that we can merge while still continuing that conversation.